assembly/std/index.ts

```diff
@@ -194,32 +303,38 @@ export function hasOpKey(key: StaticArray<u8>): bool {
 }
 
 /**
- * Get data associated with the given key from datastore
+ * Retrieves the data associated with the given key from the operation datastore.
+ *
+ * @param key - the serialized key to look for in the datastore.
  *
- * @param key -
+ * @returns - the serialized data associated with the given key as a byte array.
+ *
+ * @throws
+ * - if the key is not present in the datastore.
  *
- * @returns - data as a byte array
  */
 export function getOpData(key: StaticArray<u8>): StaticArray<u8> {
   return env.getOpData(key);
 }
 
 /**
- * Get all keys from operation datastore
+ * Retrieves all the keys from the operation datastore.
+ *
+ * @returns - a list of keys (e.g. a list of byte array)
  *
- * @returns - a list of key (e.g. a list of byte array)
  */
 export function getOpKeys(): Array<StaticArray<u8>> {
```

Comment:
Since the function getKeys and getKeysOf are doing the exact same has this function but with optional parameter in addition:  is this function to be marked deprecated ?

Reply:
We can live with this for now as it could be a clue when reading the smart contract code that we want the actual keys.